action.yml

```diff
@@ -0,0 +1,37 @@
+name: 'benchmarks-action'
+description: 'Run Artipie Benchmarks Github action'
+inputs:
+  target:
+    description: 'Name of repository (keys from adapters/benchmarks.json)'
+    required: true
+  tag:
+    description: 'Release tag for which benchmarks will be run'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/checkout@v2.3.3
```

Comment:
`actions/checkout` is usually not needed for custom action because it's performed by the workflow step:
```yaml
step:
  - uses: actions/checkout@2.3.3
  - uses: artipie/benchmarks@latest
```